jboss_wildfly/README.md

```diff
@@ -0,0 +1,57 @@
+# Agent Check: JBoss/WildFly
+
+## Overview
+
+This check monitors [JBoss][1] and [WildFly][2] applications.
+
+## Setup
+
+### Installation
+
+The JBoss/WildFly check is included in the [Datadog Agent][3] package.
+No additional installation is needed on your server.
+
+### Configuration
+
+1. Edit the `jboss_wildfly.d/conf.yaml` file, in the `conf.d/` folder at the root of your
+   Agent's configuration directory to start collecting your JBoss or WildFly application server's
+   performance data. See the [sample jboss_wildfly.d/conf.yaml][3] for all available configuration options.
+
+   This check has a limit of 350 metrics per instance. The number of returned metrics is indicated in the info page.
+   You can specify the metrics you are interested in by editing the configuration below. 
+   To learn how to customize the collected metrics, visit the [JMX Checks documentation][4] for more detailed instructions.
+   If you need to monitor more metrics, contact [Datadog support][8].
```

Comment:
General cuestion, do we give support heads up on these things or just let them come to us when they get the request?